assets/locales/cs.json

```diff
@@ -267,16 +268,16 @@
     "PRINT": "Tisk",
     "ROUTING": "Směrování",
     "SAVECOMPOSITION": "Uložit aktuální mapu",
-    "SEARCH": "Vyhledávání",
+    "SEARCH": "Hledání",
     "SENSORS": "Senzory",
     "TRACKING": "Sledování",
-    "TRIP_PLANNER": "Trip planner"
+    "TRIP_PLANNER": "Plánovač výletů"
```

Comment:
More commont would probably be "Plánovač tras"

Reply:
I am still thinking how to distinguish between "trip", "route" and "track". And whether we need to strictly distinguish them or use them interchangeably.

Reply:
I would not distinguish.